vowpalwabbit/simple_label.h

```diff
@@ -11,17 +11,16 @@ struct vw;
 struct label_data
 {
   float label = 0.f;
-  // only used for serialization and parsing.  example.weight is used for
-  // computation
-  // DeSerialized/Parsed values are copied into example in VW::setup_example()
-  float serialized_weight = 0.f;
```

Comment:
This will change the ABI. Is that ok? Do we ever pass labels across an API layer?

Reply:
It will change the ABI, not sure if that matters. But this splitting of label into label+reduction_features actually will change the API is a stronger way and should probably be reworked and trigger a major bump as we work out how the external API deals with that change